recipe/meta.yaml

```diff
@@ -10,17 +10,17 @@ source:
 
 build:
   noarch: python
-  number: 0
+  number: 1
   script: {{ PYTHON }} -m pip install . --no-deps -vv
 
 requirements:
   host:
-    - python >=3.6
+    - python >=3.7
     - pip
     - setuptools_scm
 
   run:
-    - python >=3.6
+    - python >=3.7
     - numpy >=1.17
     - pandas >=0.25
```

Comment:
Do you mind also bumping the pandas and setuptools dependencies?

- Python (3.7 or later)
- setuptools (40.4 or later)
- numpy (1.17 or later)
- pandas (1.0 or later)

Reply:
Thanks, done except for `numpy`, which was already at 1.17